Makefile

```diff
@@ -7,12 +7,17 @@ OCAML_LIB_DIR=$(shell ocamlc -where)
 
 CTYPES_LIB_DIR=$(shell ocamlfind query ctypes)
 
-OCAMLBUILD=CTYPES_LIB_DIR=$(CTYPES_LIB_DIR) OCAML_LIB_DIR=$(OCAML_LIB_DIR) \
-	ocamlbuild -use-ocamlfind -classic-display
+LWT_LIB_DIR=$(shell ocamlfind query lwt)
+
+OCAMLBUILD=CTYPES_LIB_DIR=$(CTYPES_LIB_DIR) \
+           LWT_LIB_DIR=$(LWT_LIB_DIR)       \
+           OCAML_LIB_DIR=$(OCAML_LIB_DIR)   \
+             ocamlbuild -use-ocamlfind -classic-display
 
 WITH_UNIX=$(shell ocamlfind query \
             ctypes unix unix-type-representations unix-errno.unix \
             > /dev/null 2>&1 ; echo $$?)
+WITH_LWT=$(shell ocamlfind query lwt unix > /dev/null 2>&1 ; echo $$?)
```

Comment:
I'm not very fond of this build system magic. This make the life harder to opam users to know exactly which depots needs to be installed to enable the given feature.

(this is just a side-node, I have unfortunately no better proposal, although passing env variables to `make` could also be a solution)